src/app/ecosystem-explorer/project-form/services/github/api.ts

```diff
@@ -0,0 +1,121 @@
+import { Octokit } from '@octokit/rest'
+
+// https://github.com/octokit/octokit.js
+// https://octokit.github.io/rest.js/v20/
+const octokit = new Octokit({ auth: process.env.GITHUB_AUTH_TOKEN })
+
+type SHA = string
+
+type Tree = {
+  path: string
+  mode: '100644'
+  type: 'blob'
+  sha: SHA
+}
+
+const owner = 'FilecoinFoundationWeb'
+const repo = 'filecoin-foundation'
+const baseBranch = 'main'
```

Comment:
should we use capitalised variables?

Reply:
Yes - Also feel like we might want to add these to a constants file somewhere.

Reply:
I would usually agree with you, but because the GitHub client expects lowercase variable names, it's easier this way.

Reply:
I already changed it in a CR 😊